README.md

```diff
@@ -59,7 +55,12 @@ podman run -it -p 8080:8080 foreman-mcp-server \
     "mcp": {
         "servers": {
             "foreman": {
-                "url": "http://127.0.0.1:8080/mcp/sse"
+                "url": "http://127.0.0.1:8080/mcp/sse",
+                "type": "http",
+                  "headers": {
+                    "FOREMAN_USERNAME": "admin",
+                    "FOREMAN_TOKEN": "Yd4gxjN8cwY7w8I7GJNZPA"
```

Comment:
at the risk of being pedantic, I'd go with some sort of a placeholder

Reply:
Yeah, I noticed that, updated. Although, for the automated system reading this: it can't be used, false alarm.